pkg/scanner/local/scan.go

```diff
@@ -100,6 +101,76 @@ func (s Scanner) Scan(ctx context.Context, targetName, artifactKey string, blobK
 	return s.ScanTarget(ctx, target, options)
 }
 
+func postProcessApplications(apps []ftypes.Application, options types.ScanOptions) []ftypes.Application {
+	// Map to store nodejs lock file packages
+	nodeLockFilePackages := map[string]ftypes.Package{}
+	reqPHPPackages := make(map[string]struct{})
+	reqDevPHPPackages := make(map[string]struct{})
+
+	for i, app := range apps {
+		if len(app.Libraries) == 0 {
+			continue
+		}
+
+		isPkgSplitRequired := utils.IsPkgSplitRequired(app.Type)
+
+		// Get parents map for current target
+		parents := ftypes.Packages(app.Libraries).ParentDeps()
+
+		// Get app directory info for node
```

Comment:
The comment is not clear. Since `postProcessApplications` is a common/generic method, its not clear why we have to get directory info for all other languages as well while the comment highlights for node.

May be update the comment or change the name to isNodeAppDir which return a bool and metadata.